pkg/kubelet/util/manager/cache_based_manager.go

```diff
@@ -24,26 +24,28 @@ import (
 	"time"
 
 	"k8s.io/api/core/v1"
-	storageetcd3 "k8s.io/apiserver/pkg/storage/etcd3"
-	"k8s.io/kubernetes/pkg/kubelet/util"
-
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/clock"
 	"k8s.io/apimachinery/pkg/util/sets"
+	storageetcd3 "k8s.io/apiserver/pkg/storage/etcd3"
+	"k8s.io/kubernetes/pkg/kubelet/kubeclientmanager"
+	"k8s.io/kubernetes/pkg/kubelet/util"
 )
 
 // GetObjectTTLFunc defines a function to get value of TTL.
 type GetObjectTTLFunc func() (time.Duration, bool)
 
 // GetObjectFunc defines a function to get object with a given tenant, namespace and name.
```

Comment:
New argument added, need to give name or update comment

Reply:
will add based on feedback